Tasks/InstallAppleCertificate/task.json

```diff
@@ -11,7 +11,7 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 1,
+        "Major": 2,
```

Comment:
I'm bumping the major version because requiring a password for the default keychain is a breaking change.

Reply:
Should the `certPwd` parameter be required now?

Reply:
No, you can supply a p12 that has no password.

If you mean `keychainPwd`, it's optional for temporary keychains.

Reply:
Oh yes, I suppose I did mean that.  Thanks.

Reply:
Switched to making keychainPassword required, and hiding it when `Temporary Keychain` is selected.